packages/commonwealth/server/routes/updateCommunityCategory.ts

```diff
@@ -38,6 +44,45 @@ const updateCommunityCategory = async (
     );
   });
 
+  const { tag_ids } = req.body;
+  if (tag_ids) {
+    // verify all tag ids exist
+    if (tag_ids.length > 0) {
+      const tagCount = await models.Tags.count({
+        where: {
+          id: tag_ids,
```

Comment:
I think this needs to be an in query, something like:

`id: { [Op.in]: tag_ids }`